registry/domain/services/organization_service.py

```diff
@@ -30,20 +32,12 @@ def publish_org(self, org_uuid, username):
         if len(orgs) == 0:
             raise Exception(f"Organization not found with uuid {org_uuid}")
         organization = orgs[0]
-        metadata = organization.to_metadata()
-        metadata_ipfs_hash = self._publish_org_to_ipfs(metadata, org_uuid)
-        organization.set_metadata_ipfs_hash(metadata_ipfs_hash)
+        organization.publish_assets()
```

Comment:
can we club them into one function publish and internally handle in two function